ADR/0048-movable-build-tests.md

```diff
@@ -0,0 +1,68 @@
+# 48. Attestable Build-Time Tests in Integration Service
+
+Date: 2025-04-06
+
+## Status
+
+Proposed
+
+## Context
+
+Some of the security scans run during the build pipeline take a long time to
+run. This forces users to wait longer to be notified of the results of their
+builds.  In order to receive more immediate feedback, users have requested that
+they be able to run security scans during their integration tests.
+
+## Decision
+
+We will add support for adding attestations to images from their integration
+pipelines. Test pipelines can be run and re-run independently of build
+pipelines but all pipelines will have attestations generated for them to enable
+Conforma evaluations.
+
+### Attestations
+
+Each task will be responsible for generating an attestation in Cosign. `vuln`
+attestations will be used for security scans rather than `provenance`
+attestations. Tasks will output `ARTIFACT_URI` and `ARTIFACT_DIGEST` which will
+point to the location of the attestation in Quay.  These results will tell
+Chains to sign the attestation.
```

Comment:
Just want to be clear that chains won't sign the attestation directly generated by the task.

```suggestion
point to the location of the attestation in Quay.  These results will tell
Chains to generate and sign a SLSA provenance attestation for the
task-generated attestation.
```

Reply:
I'll change that, but doesn't Chains generate a [vulnerability attestation](https://github.com/in-toto/attestation/blob/main/spec/predicates/vulns_02.md)? Or does that attestation type only apply to the task-generated attestation and the Chains-generated attestation is always a SLSA provenance attestation?